src/style.css

```diff
@@ -0,0 +1,26 @@
+* {
+  margin: 0;
+  padding: 0;
+  border: 0;
```

Comment:
do not use * to reset margin, paddings and so on

specify elements that you need to reset styles for